core/src/main/scala/com/mesosphere/usi/core/Response.scala

```diff
@@ -0,0 +1,34 @@
+package com.mesosphere.usi.core
+
+import com.mesosphere.usi.models.{Mesos, StatusMessage}
+
+sealed trait Response {
```

Comment:
What is a response?

Reply:
yeah I have problems with naming in this part as well... :/ I think we should find better name for this class